ChiModules/LinearBoltzmannSolver/lbs_compute_precursors.cc

```diff
@@ -0,0 +1,61 @@
+#include "lbs_linear_boltzmann_solver.h"
+
+#include "ChiMath/SpatialDiscretization/FiniteElement/PiecewiseLinear/pwl.h"
+
+#include "chi_log.h"
+extern ChiLog& chi_log;
+
+
+//###################################################################
+/**Compute the steady state delayed neutron precursor concentrations.*/
+void LinearBoltzmann::Solver::ComputePrecursors()
+{
+  auto pwl =
+      std::dynamic_pointer_cast<SpatialDiscretization_FE>(discretization);
```

Comment:
Since we are not casting to `SpatialDiscretization_PWL`, can we call this variable `fe` instead of `pwl`.

Reply:
Technically at the moment everything is `pwl` since only linear elements are supported, right (I.e. `pwld` and `pwlc` fall under `pwl`)? For anticipating future support for higher order elections, I agree with this change though.